Source/Krypton Components/Krypton.Ribbon/General/CalculatedValues.cs

```diff
@@ -79,6 +79,7 @@ public void Recalculate()
                     case PaletteRibbonShape.Microsoft365:
                     case PaletteRibbonShape.VisualStudio:
                     case PaletteRibbonShape.Office2010:
+                    case PaletteRibbonShape.VisualStudio2010:
```

Comment:
I thought that I saw the palette called vs20**22**

Reply:
The `VisualStudio2010` will be for the 2010 style (which will be based off O2k10) & `VisualStudio` will be based off the MS365 themes with colour variations between 2012 - 2022.